### DISCUSSION TITLE
Column alignment in p-table is gets distorted after unchecked all the columns from column chooser.

### DISCUSSION THREAD

**Discussion by pk2018student** [primefaces/community#814] [PrimeNG]
![Screenshot (560)](https://github.com/primefaces/community/assets/155537282/d7d990af-a7d7-49b4-8e6d-12285aafee19)
Original p-tabel 
After unchecking optional column from column chooser 
![Alignment](https://github.com/primefaces/community/assets/155537282/9b12e95e-41a0-449b-8131-8794a63f2166)
I want to have column-chooser icon p-header and p-header of copy icon should have fixed width irrespective of change in number of coulmns

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Hi,

Closing the topic due to the inactivity.
